src/EnergyPlus/MixedAir.cc

```diff
@@ -3279,6 +3275,11 @@ namespace MixedAir {
 						SetupEMSActuator( "Outdoor Air Controller", OAController( OAControllerLoop ).Name, "Air Mass Flow Rate", "[kg/s]", OAController( OAControllerLoop ).EMSOverrideOARate, OAController( OAControllerLoop ).EMSOARateValue );
 					}
 
+					if ( OAController( OAControllerLoop ).VentMechObjectNum > 0 && thisOASys > 0){
+						if ( !VentilationMechanical( OAController( OAControllerLoop ).VentMechObjectNum ).DCVFlag ){
+							AirLoopControlInfo( thisOASys ).AirLoopDCVFlag = false;
```

Comment:
@JasonGlazer Everything that's inside the two `if ( AirLoopZoneInfoSize >= AirLoopNum )` blocks.